sympy/physics/quantum/density.py

```diff
@@ -14,7 +14,7 @@
 from sympy.physics.quantum.matrixutils import numpy_ndarray, scipy_sparse_matrix, to_numpy
 from sympy.physics.quantum.tensorproduct import TensorProduct, tensor_product_simp
 from sympy.physics.quantum.trace import Tr
-
+from sympy import conjugate 
```

Comment:
I think you need to get rid of the trailing whitespace on this line for the code quality check to pass

Reply:
Ok I understood thank you.